sklearn/pipeline.py

```diff
@@ -272,6 +278,22 @@ def inverse_transform(self, X):
         return Xt
 
     @if_delegate_has_method(delegate='_final_estimator')
```

Comment:
No, it's applicable if *any* of the estimators supports `pipe` and the last supports `transform` or `pipe`.